web/src/lightbox.js

```diff
@@ -491,11 +491,11 @@ export function parse_media_data(media) {
     if (is_compose_preview_media) {
         sender_full_name = people.my_full_name();
     } else {
-        const $message = $parent.closest("[zid]");
-        const zid = rows.id($message);
-        const message = message_store.get(zid);
+        const $message = $parent.closest(".message_row");
+        const message_id = rows.id($message);
```

Comment:
Should this part be using something from the `rows` module?

Reply:
We have this method:
```js

export function get_closest_row(element: string): JQuery {
    return $(element).closest("div.message_row");
}

```

but I am not sure if the refactor is worth it.

Reply:
Yeah that function works here. It's not a huge number of references, so we can just leave this as a fun refactor for later. I think it is nice to have as little code directly reading the `message_row` DOM as possible.

Reply:
Okay, added a commit for that refactor.